src/app/home/home.page.scss

```diff
@@ -18,15 +18,15 @@
   }
 
   &.activities {
-    background-color: #56c4cf;
+    background-color: #fbad18;
   }
 
   &.chat {
-    background-color: #e7406d;
+    background-color: #fbad18;
   }
 
   &.partners {
-    background-color: #6e78b9;
+    background-color: #fbad18;
```

Comment:
@estherwafula  why are these changed to same colour? Only the chat was to be changed.